### PR TITLE
Tml 32609 create a new java package for translation improvement

### PR DESCRIPTION
**Pull request by Cloudwalker1986**

### CODE REVIEW

**Review by fdiazarce — APPROVED** [review]

File: sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java [hunk 1]
```diff
@@ -1,52 +1,98 @@
...
+    String line = reader.readLine();
+    while (line != null) {
+      sb.append(line);
+      line = reader.readLine();
     }
+    return sb.toString();
+  }
+
+  private CloseableHttpClient getHttpClient() throws URISyntaxException {
+
```
Comment (aburlacu) [inline sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java:91]: maybe you can remove empty line

File: sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java [hunk 2]
```diff
@@ -1,52 +1,98 @@
...
-        String line = reader.readLine();
-        while (line != null) {
-            sb.append(line);
-            line = reader.readLine();
-        }
-        return sb.toString();
+  private String toJsonString(BufferedReader reader) throws IOException {
+    StringBuilder sb = new StringBuilder();
+    String line = reader.readLine();
+    while (line != null) {
```
Comment (aburlacu) [inline sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java:83]: it make sense to check  if isEmpty?

**Review by aburlacu — APPROVED** [review]

File: sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java (hunk 1 above)
Comment (Cloudwalker1986) [reply sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java:91]: is automated added by the tool for code style which is used for consignment service already.

File: sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java (hunk 2 above)
Comment (Cloudwalker1986) [reply sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java:83]: Empty is some time expected ?! ;) Even so, this code of line is used in all java Packages.

**Pull request merged**